auth-web/src/components/auth/manage-business/AddNameRequestForm.vue

```diff
@@ -5,12 +5,12 @@
         <legend hidden>Name Request Number and Applicant Phone Number or Email Address</legend>
         <v-text-field
           ref="nrNumber"
-          filled persistent-hint validate-on-blur
+          filled persistent-hint
           label="Enter a Name Request Number"
           hint="Example: NR 1234567"
           :rules="nrNumberRules"
           v-model="nrNumber"
-          @blur="formatNrNumber()"
+          @keyup="uppercase('nrNumber')"
```

Comment:
Softly typed strings are fairly prone to errors

Reply:
It is the same requirement as in business-edit-ui, which is ticket 14438 for.